nservicebus/messaging/transactions.md

```diff
@@ -23,9 +23,11 @@ Based on transaction handling mode, NServiceBus offers three levels of guarantee
 
 ### Unreliable
 
-In this mode the transport in use does not attempt to wrap the receive operation in any kind of transaction. Once the message has been received, it can't be put back into the queue. Should the message processing fail for any reason (including system crash), the message is **permanently lost**. 
+In this mode the transport in use does not attempt to wrap the receive operation in any kind of transaction. 
 
-NOTE: No retries will be performed in this mode and messages will not be forwarded to the error queue in case of failures
+Once a message has been received, if the message processing fails it should still be put into the error queue. If there is a critical failure, including system or endpoint crash, the message is **permanently lost**. 
```

Comment:
> Document current behavior as is with what we intend to do in V6. 

Nope. if you do that and merge the PR then we have doco that is the wrong tense and becomes historically incorrect once we release v6. It should be document is a way that is unambiguous the version. eg `version 5 and below` and `Version 6 and higher`. this way we can merge and not need to revisit later.